hub3/server/http/handlers/search.go

```diff
@@ -163,14 +163,15 @@ func ProcessSearchRequest(w http.ResponseWriter, r *http.Request, searchRequest
 		return
 	}
 
-	pager, err := searchRequest.NextScrollID(res.TotalHits())
+	pager, err := searchRequest.ScrollPagers(res.TotalHits())
 	if err != nil {
 		log.Println("Unable to create Scroll Pager. ")
 		return
 	}
 
 	// Add scrollID pager information to the header
-	w.Header().Add("P_SCROLL_ID", pager.ScrollID)
```

Comment:
The P_SCROLL_ID is renamed to P_NEXT_SCROLL_ID. This may breaking existing implementations that rely on this header.

Reply:
Currently, there are no implementations that rely on this.